packages/agent-toolkit/src/core/tools/platform-api-tools/workforms-tools/create-form-tool.ts

```diff
@@ -0,0 +1,64 @@
+import { z } from 'zod';
+import {
+  BoardKind,
+  CreateFormMutation,
+  CreateFormMutationVariables,
+} from '../../../../monday-graphql/generated/graphql';
+import { createForm } from './workforms.graphql';
+import { ToolInputType, ToolOutputType, ToolType } from '../../../tool';
+import { BaseMondayApiTool, createMondayApiAnnotations } from '../base-monday-api-tool';
+import { GraphQLDescriptions } from './workforms.consts';
+
+export const createFormToolSchema = {
+  destination_workspace_id: z.number().describe(GraphQLDescriptions.form.args.destinationWorkspaceId),
```

Comment:
this should be of type string, if in the gql its number this should change. I know someone from your team plans to change it to type ID (for all the IDS)

Reply:
This is dependent on the use_template mutation so we copy the exact types they use as well. When they switch to IDs we will too